src/components/InputComponent/inputComponent.types.d.ts

```diff
@@ -0,0 +1,24 @@
+import { HTMLAttributes } from 'react';
+
+enum Rounded {
+  rounded = 'rounded',
+  normal = 'normal',
+}
+
+/**
+ * Types for Input
+ * @param {string} border - Set border radius
+ * @param {boolean} dark - Dark theme
+ * @param {boolean} disabled - Disable input
+ * @param {string} error - Set the error
+ * @param {function} inputValueReciever - Function that recieves input value
+ */
+
+export interface IInputProps extends HTMLAttributes<HTMLInputElement> {
+  border: Rounded.normal | Rounded.rounded;
+  dark: boolean;
+  disabled: boolean;
```

Comment:
Isn't it included int he `HTMLInputElement` interface?

Reply:
https://github.com/igorizviekov/mt-booking_app/pull/24/commits/74ff661575e673759d0bc1baf2f28de96e5ddb28